manifests/dev/vsphere-7.0u2/vanilla/rbac/vsphere-csi-node-rbac.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: vsphere-csi-node
   namespace: kube-system
 ---
-kind: ClusterRole
+kind: Role
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: vsphere-csi-node-cluster-role
```

Comment:
Could you also change the name to `vsphere-csi-node-role` everywhere?

Reply:
After merging the rules, looks like `vsphere-csi-node-cluster-role` can be removed. Take a look at the latest commit